cmd/esc/cli/env_rollback.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023, Pulumi Corporation.
+
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
+	"github.com/spf13/cobra"
+)
+
+func newEnvRollbackCmd(env *envCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rollback [<org-name>/]<environment-name>@<version>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Roll back to a specific version",
+		Long: "Roll back to a specific version\n" +
+			"\n" +
+			"This command rolls an environment's definition back to the specified\n" +
+			"version. The environment's definition will be replaced with the\n" +
+			"definition at that version, creating a new revision.\n",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+
+			if err := env.esc.getCachedClient(ctx); err != nil {
+				return err
+			}
+
+			orgName, envName, version, args, err := env.getEnvName(args)
+			if err != nil {
+				return err
+			}
+			if version == "" {
+				return errors.New("please specify a version")
+			}
+			_ = args
```

Comment:
Any reason to have this as a separate line instead of replacing `args` with `_` on line 32?

Reply:
Yes. I felt that this was clearer if this code ever changes s.t. it reads the args after the environment name is consumed.